com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -181,6 +181,27 @@ public void EnableAutomaticStepping()
             m_FixedUpdateStepper = m_StepperObject.AddComponent<AcademyFixedUpdateStepper>();
         }
 
+        /// <summary>
+        /// Registers SideChannel to the Academy to send and receive data with Python.
+        /// If IsCommunicatorOn is false, the SideChannel will not be registered.
+        /// </summary>
+        /// <param name="sideChannel"> The side channel to be registered.</param>
+        public void RegisterSideChannel(SideChannel channel)
+        {
+            LazyInitialization();
+            Communicator?.RegisterSideChannel(channel);
+        }
+
+        /// <summary>
+        /// Deregisters SideChannel to the Academy. If the side channel was not registered, 
+        /// nothing will happen.
+        /// </summary>
+        /// <param name="sideChannel"> The side channel to be deregistered.</param>
+        public void DeregisterSideChannel(SideChannel channel)
```

Comment:
nit: I've never seen the use of `Deregister` before.  I'm not against it, just curious between the difference between prefix `De` and `Un`.

Reply:
Seems to be the same. I will change to Unregister since it is more common.